doc/source/programs/gdalinfo.rst

```diff
@@ -196,6 +196,8 @@ Example
     Center      (  456080.000, 3735960.000) (117d28'27.39"W, 33d45'52.46"N)
     Band 1 Block=512x16 Type=Byte, ColorInterp=Gray
 
+For DD coordinates instead of DMS, inspect the wgs84Extent member of gdalinfo -json:
+
```

Comment:
That sounds like the json format exists mainly for showing extent in DD, while I think that the main advantage is that json is computer-friendly.
I wonder if there is a good reason for DMS, or if decimal degrees could be used always.

Reply:
Yeah I kind of "just happened to squeeze the sentence into the right spot", " perhaps giving that effect."

On the other hand we don't want to get rid of DMS completely. Maybe some people still like it. Gives sort of like that early mariners feeling. Good for historical publishing. 

Anyway I just wanted the man page to sort of say something for people who are baffled why didn't the authors ever think of using DD format.

But you're entirely welcome to change it anyway you like because I'm sure you'll come up with a better way than me. 

I just didn't want to get my hands dirty to do any actual programming.